.gitignore

```diff
@@ -5,4 +5,7 @@
 dist
 **/*.html
 !test/index.html
-test/index.json
\ No newline at end of file
+test/index.json
+
+# built js
+src/vastly.*
```

Comment:
Build artifacts tend to go in `/dist`, `/src` (as the name indicates) is for source files.